jsonmodels/fields.py

```diff
@@ -20,13 +25,17 @@ def __init__(
             nullable=False,
             help_text=None,
             validators=None,
-            default=None):
+            default=NO_DEFAULT):
         self.memory = WeakKeyDictionary()
         self.required = required
         self.help_text = help_text
         self.nullable = nullable
-        self._default = default
         self._assign_validators(validators)
+        self.has_default = default is not NO_DEFAULT
+        default = default if self.has_default else None
```

Comment:
You could go with those properties like I mentioned above - this would simplify code.

Reply:
fixed